setup-scripts/health-checker.py

```diff
@@ -211,59 +232,118 @@ async def check_health():
-        return await send_msg(
-            client, "Failed to run the checks!", file=trace, force_notify=True
-        )
+    # Execute siac alerts and read the response
+    cmd_string = "docker exec {} siac alerts".format(CONTAINER_NAME)
+    siac_alert_output = os.popen(cmd_string).read().strip()
 
-    alerts =  alerts_json['alerts']
-    critical_alerts =  alerts_json['criticalalerts']
-    error_alerts =  alerts_json['erroralerts']
-    warning_alerts =  alerts_json['warningalerts']
+    # Initialize variables
+    num_critical_alerts = 0
+    num_error_alerts = 0
+    num_warning_alerts = 0
+    num_siafile_alerts = 0
     siafile_alerts = []
-    siafile_alert_message = "The SiaFile mentioned in the 'Cause' is below 75% redundancy"
 
-    # Check for siafile alerts in alerts. This is so that the alert severity
-    # can change and this doesn't need to be updated
-    for alert in alerts:
-        if alert['msg'] == siafile_alert_message:
-            siafile_alerts.append(alert)
+    # Pattern strings to search for
+    critical = 'Severity: critical'
+    error = 'Severity: error'
+    warning = 'Severity: warning'
+    health_of = 'has a health of'
+    siafile_alert_message = "The SiaFile mentioned in the 'Cause' is below 75% redundancy"
 
+    # Split the output by line and check for type of alert and siafile alerts
+    for line in siac_alert_output.split("\n"):
+        # Check for the type of alert
+        if contains_string(line, critical):
+            num_critical_alerts += 1
+        if contains_string(line, error):
+            num_error_alerts += 1
+        if contains_string(line, warning):
+            num_warning_alerts += 1
+
+        # Check for siafile alerts in alerts. This is so that the alert
+        # severity can change and this doesn't need to be updated
+        if contains_string(line, siafile_alert_message):
+            num_siafile_alerts += 1
+        if contains_string(line, health_of):
+            siafile_alerts.append(line)
 
     ################################################################################
-    ################ create a message
+    # create a message
     ################################################################################
 
     message = ""
     force_notify = False
 
-    if len(critical_alerts) > 0:
-        message += "{} CRITICAL Alerts found! ".format(len(critical_alerts))
-        force_notify = True
-    if len(error_alerts) > 0:
-        message += "{} Error Alerts found! ".format(len(error_alerts))
+    if num_critical_alerts > 0:
+        message += "{} CRITICAL Alerts found! ".format(num_critical_alerts)
         force_notify = True
-    
-    message += "{} Warning Alerts found. ".format(len(warning_alerts))
-    message += "{} SiaFiles with bad health found. ".format(len(siafile_alerts))
+    if num_error_alerts > 0:
+        message += "{} Error Alerts found! ".format(num_error_alerts)
+        force_notify = num_error_alerts > num_siafile_alerts
```

Comment:
I would remove this line. It doesn't make sense. There is currently only 1 error alert in the whole code base. Even if we add more error alerts I don't think this makes sense, we should either alert on error alerts or not. I think it is OK to not alert and just send it in a message.

Reply:
removed